src/main/java/org/thoughtcrime/securesms/qr/QrScanFragment.java

```diff
@@ -106,8 +106,11 @@ public MyCaptureManager(Activity activity, DecoratedBarcodeView barcodeView) {
         // which makes _showing_ the QR-code impossible if scanning goes wrong.
         // therefore, we only show a non-disturbing error here.
         @Override
-        protected void displayFrameworkBugMessageAndExit() {
-            Toast.makeText(myActivity, R.string.zxing_msg_camera_framework_bug, Toast.LENGTH_SHORT).show();
+        protected void displayFrameworkBugMessageAndExit(String message) {
+            if (message == null || message.isEmpty()) {
```

Comment:
good catch with the null pointer. the following is shorter, but less explicit, a matter of taste i'd say :)

```suggestion
            if (TextUtils.isEmpty(message)) {
```